circle.yml

```diff
@@ -1,11 +1,15 @@
-machine:
-    services:
-        - docker
-
-dependencies:
-    override:
-        - echo "Ignore CircleCI detected dependencies"
-
-test:
-    override:
-        - docker build -t steemit/steemit.com .
+version: 2
+jobs:
+    build:
+        docker:
+            - image: circleci/openjdk:jdk-node-browsers
```

Comment:
Only docker is needed here, I suggest using `docker:17`

Reply:
10 4; done